client/public/css/home.css

```diff
@@ -130,6 +130,7 @@ New color scheme reference is in utils.css
 #home-bg {
   background: url('../img/new-hero.png') center center;
   min-height: 100vh;
+  height: 100px;
```

Comment:
why 100px?

Reply:
That is a hack i found to make IE11 work when justifying content and aligning items center. It will still use the min height bUT needs a height property.

Reply:
I had no idea! that's cool